### PR TITLE
crio: finish adding core user to swap and serial jobs

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt~ <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
/hold

quick typo

File: config/jobs/kubernetes/sig-node/node-kubelet.yaml [hunk 1]
```diff
@@ -184,7 +185,6 @@ periodics:
           - --scenario=kubernetes_e2e
           - --
           - --deployment=node
-          - --env=KUBE_SSH_USER=core
```
Comment (haircommander) [inline config/jobs/kubernetes/sig-node/node-kubelet.yaml:187]: this one is an ubuntu one so doesn't need core

Comment (harche) [reply config/jobs/kubernetes/sig-node/node-kubelet.yaml:187]: Agree

**Comment by haircommander** [comment]
/hold cancel

okay ready

**Comment by haircommander** [comment]
PTAL @SergeyKanzhelev @mrunalp

**Review by mrunalp — APPROVED** [review]

**Pull request merged**